packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -0,0 +1,272 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-clip: padding-box; // Stops background color from bleeding through border on IE.
+    border: var(--terra-avatar-border, solid 1px #fff);
```

Comment:
We should update these variables to match the new naming convention

Reply:
updated https://github.com/cerner/terra-core/pull/2061/commits/1a40018162ead66c575f6adb2a3b2167a2f9b0b7#diff-807bd069cd8c6ba2acfbc93c8ecb787eL6